compiler/rustc_codegen_cranelift/scripts/config.sh

```diff
@@ -1,3 +1,4 @@
+#!/bin/sh
```

Comment:
Is this possible to run in posix shell? I see `[[` which I think it is part of bash.